src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -331,7 +331,7 @@ type UseBalanceChartData = (args: UseBalanceChartDataArgs) => UseBalanceChartDat
 */
 export const useBalanceChartData: UseBalanceChartData = args => {
   const { assetId, accountId, timeframe } = args
-  const assets = useAppSelector(selectAssets)
+  const assets = useAppSelector(selectFungibleAssets)
```

Comment:
Even though we currently don't have market data per se for non-fungible, should we assume we can get *some* kind market data for non-fungible? Which we do as floor price in the NFT tab, even though we don't upsert it as market data:

<img width="303" alt="image" src="https://github.com/shapeshift/web/assets/17035424/d00cb7c7-3741-4293-a724-0de1abd41492">

See Zerion for instance, which do exclude it from the net worth, but still have a tab for NFTs with total floor price value

<img width="254" alt="image" src="https://github.com/shapeshift/web/assets/17035424/eb462638-8ce6-4a36-8457-a7bc845159e0">